File: Easy/2678. Number of Senior Citizens/main.js
const test = [
    ["7868190130M7522","5303914400F9211","9273338290F4010"],
    ["1313579440F2036","2921522980M5644"]
]

var countSeniors = function(details) {
    let count = 0;
    for(let num of details) {
        const findAge = Number(num.substr(11, 2));
        (findAge > 60) && count++;
    }
    return count;
};

test.map((cases) => console.log(countSeniors(cases)));

File: Easy/2678. Number of Senior Citizens/main.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	tests := [][]string{
		{ "7868190130M7522","5303914400F9211","9273338290F4010" },
		{ "1313579440F2036","2921522980M5644"},
	}

	for _, test := range tests {
		fmt.Println(countSeniors(test))
	}
}

func countSeniors(details []string) int {
	count := 0
    for _, str := range details {
		num , err := strconv.Atoi(str[11:13])
		if err == nil && num > 60 {
			count++
		}
	}
	return count
}